### PR TITLE
Update rtl8720e/loadable_apps configuration

### PR DESCRIPTION
**Pull request by thapav**

### CODE REVIEW

File: build/configs/rtl8720e/loadable_apps/defconfig [hunk 1]
```diff
@@ -214,7 +214,7 @@ CONFIG_BOOT_RUNFROMFLASH=y
 #
 # Boot Memory Configuration
 #
-CONFIG_RAM_KREGIONx_START="0x20080020,0x60000020,0x60100020,"
+CONFIG_RAM_KREGIONx_START="0x20080020,0x60000020,0x60101020,"
```
Comment (abhishek-samsung) [inline build/configs/rtl8720e/loadable_apps/defconfig:217]: Could you please explain why this change is needed? 0x60000020 + 1048576 = 0x60100020 so why does overlap happen? And if overlap happens should we change in 8721 also?

https://github.com/Samsung/TizenRT/blob/253b47dfbf3ea1f04f55bb4ff34d4296af6a57a8/build/configs/rtl8721csm/loadable_apps/defconfig#L212-L213

File: build/configs/rtl8720e/loadable_apps/defconfig [hunk 2]
```diff
@@ -266,9 +266,9 @@ CONFIG_ARCH_BOARD_HAVE_FLASH=y
 #
 CONFIG_FLASH_PARTITION=y
 CONFIG_FLASH_MINOR=0
-CONFIG_FLASH_PART_SIZE="32,28,16,4,432,2556,2556,512,512,512,512,512,8,"
-CONFIG_FLASH_PART_TYPE="none,none,none,none,none,kernel,kernel,smartfs,bin,bin,bin,bin,bootparam,"
-CONFIG_FLASH_PART_NAME="bl1,reserved,ftl,system_data,ss,kernel,kernel,userfs,app1,app1,app2,app2,bootparam,"
+CONFIG_FLASH_PART_SIZE="32,28,16,4,432,2044,2044,512,512,512,512,512,1024,8,"
+CONFIG_FLASH_PART_TYPE="none,none,none,none,none,kernel,kernel,smartfs,bin,bin,bin,bin,none,bootparam,"
+CONFIG_FLASH_PART_NAME="bl1,reserved,ftl,system_data,ss,kernel,kernel,userfs,app1,app1,app2,app2,ss_reserved,bootparam,"
```
[the remark this answers is not in the document]
Comment (thapav) [reply build/configs/rtl8720e/loadable_apps/defconfig:271]: Updated

**Review by sunghan-chang — APPROVED** [review]

File: build/configs/rtl8720e/hello/defconfig [hunk 3]
```diff
@@ -258,7 +258,7 @@ CONFIG_ARCH_BOARD_HAVE_FLASH=y
 #
 # Board-Partition Options
 #
-#CONFIG_FLASH_PARTITION=y
+CONFIG_FLASH_PARTITION=y
 CONFIG_FLASH_MINOR=0
 CONFIG_FLASH_PART_SIZE="32,28,16,4,432,3068,3068,512,1024,8,"
 CONFIG_FLASH_PART_TYPE="none,none,none,none,none,kernel,kernel,smartfs,none,bootparam,"
```
Comment (sunghan-chang) [inline build/configs/rtl8720e/hello/defconfig:264]: could you rename ss_reserved to reserved here and tc as well?

File: build/configs/rtl8720e/loadable_apps/defconfig [hunk 4]
```diff
@@ -266,9 +266,9 @@ CONFIG_ARCH_BOARD_HAVE_FLASH=y
 #
 CONFIG_FLASH_PARTITION=y
 CONFIG_FLASH_MINOR=0
-CONFIG_FLASH_PART_SIZE="32,28,16,4,432,2556,2556,512,512,512,512,512,8,"
-CONFIG_FLASH_PART_TYPE="none,none,none,none,none,kernel,kernel,smartfs,bin,bin,bin,bin,bootparam,"
-CONFIG_FLASH_PART_NAME="bl1,reserved,ftl,system_data,ss,kernel,kernel,userfs,app1,app1,app2,app2,bootparam,"
+CONFIG_FLASH_PART_SIZE="32,28,16,4,432,1408,1408,152,152,1508,1508,512,1024,8,"
```
Comment (sunghan-chang) [inline build/configs/rtl8720e/loadable_apps/defconfig:269]: Let's use the same values as internal. 
```
1408,1408,1508,1508,152,152
```

File: build/configs/rtl8720e/hello/defconfig (hunk 3 above)
Comment (thapav) [reply build/configs/rtl8720e/hello/defconfig]: done

File: build/configs/rtl8720e/loadable_apps/defconfig (hunk 4 above)
Comment (thapav) [reply build/configs/rtl8720e/loadable_apps/defconfig:269]: Since we have the following app sizes on public >
KERNEL    Binary Size :  1,295,328 bytes    Partition Size :  1,441,792 bytes
APP1    Binary Size :      9,529 bytes    Partition Size :    155,648 bytes
APP2    Binary Size :    284,215 bytes    Partition Size :  1,544,192 bytes

Lets keep it as it is.

**Review by sunghan-chang — APPROVED** [review]
